server/src/main/java/org/opensearch/http/DefaultRestChannel.java

```diff
@@ -147,6 +151,14 @@ public void sendResponse(RestResponse restResponse) {
             addCustomHeaders(httpResponse, restResponse.getHeaders());
             addCustomHeaders(httpResponse, threadContext.getResponseHeaders());
 
+            // Add server header
+            serverHeader.put(
+                "serverHeader",
+                Arrays.asList("OpenSearch/" + Build.CURRENT.getQualifiedVersion() + " (" + Build.CURRENT.getDistribution() + ")")
```

Comment:
please create static strings for "serverHeader" or "OpenSearch/"

Reply:
serverHeader -> serverVersion

Reply:
Created static string and refactored server header to server version